db/access.php

```diff
@@ -27,24 +27,29 @@
 
 $capabilities = array(
     'mod/observation:view' => array(
-
         'captype' => 'read',
         'contextlevel' => CONTEXT_MODULE,
         'archetypes' => array(
             'teacher' => CAP_ALLOW,
             'editingteacher' => CAP_ALLOW,
             'student' => CAP_ALLOW,
-            'manager' => CAP_ALLOW,
         )
     ),
     'mod/observation:addinstance' => array(
         'riskbitmask' => RISK_XSS,
-
         'captype' => 'write',
         'contextlevel' => CONTEXT_MODULE,
         'archetypes' => array(
             'editingteacher' => CAP_ALLOW,
-            'manager' => CAP_ALLOW,
         )
     ),
+    'mod/observation:performobservation' => array(
+        'riskbitmask' => RISK_XSS,
```

Comment:
Does this actually have an XSS risk? IDK that there would be raw input anywhere. Perhaps personal data is a better risk marker